client/src/containers/backend/Settings/General.js

```diff
@@ -41,6 +41,12 @@ class SettingsGeneralContainer extends PureComponent {
             name="attributes[general][defaultPlaceOfPublication]"
             placeholder="Enter Default Place of Publication"
           />
+          <Form.TextInput
```

Comment:
And don't hesitate to let me know if you want help. The library we're using for input masks is a little weird.